proyect.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Proyect</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+  
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
+  
+  
+</html>
```

Comment:
- [ ] [OPTIONAL]  Good job! It is a good idea to creating this .html file according to the project requirement, but I think it will be great to rename his file index.html . When you arrive a website, for example www.website.com, you're not pointing to a file (like you would be if you typed www.website.com/about.html), you're pointing to a directory listing of all the files.

Reply:
Ok, I already fix the file name for another better one. @codecaiine